WordPress/src/main/java/org/wordpress/android/ui/bloggingreminders/BloggingRemindersUiModel.kt

```diff
@@ -2,4 +2,19 @@ package org.wordpress.android.ui.bloggingreminders
 
 import java.time.DayOfWeek
 
-data class BloggingRemindersUiModel(val siteId: Int, val enabledDays: Set<DayOfWeek> = setOf())
+data class BloggingRemindersUiModel(
+    val siteId: Int,
+    val enabledDays: Set<DayOfWeek> = setOf(),
+    val hour: Int,
+    val minute: Int
```

Comment:
We should probably use `LocalTime` here instead – it can be constructed pretty easily with `LocalTime.of(hour, minute)` and it would make formatting easier.